### PR TITLE
[odo] ocp 4.3 RN+KN

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by pmacko1** [comment]
@Preeticp @amitkrout @kadel @cdrage could you review, please?

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: The active pointer does not switch to the current active project after deletion of a project

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: the `odo watch` command...with an error...components,

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: The `...` command...variable,...

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: The `...` command ...the `...` directory.
Do we need $ in the folder name and the directory name?

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (pmacko1) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: I wanted to mention that other stuff were improved as well :)

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (pmacko1) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: ...set with `odo...?

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: ...the `--context` flag

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: The `...` command...with the `--context` and `--binary` flags

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: is listed

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: Please change to this format in all issues that start with a command: The `...` command

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: a project

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (pmacko1) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: it is only a level of verbosity, not accessing the logs themselves

Comment (pmacko1) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: but rewritten, via shouldnt be there :)

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: ...the `...` command does...

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: missing period on line 79 and 81

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (pmacko1) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: has been improved

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: an environment variable... after the first...push, then the route URL does not

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (pmacko1) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: also push timeout for what?

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (pmacko1) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: also what is Service catalog?

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: comma after cluster
the `component create` command?
an error
throws/displays
the catalog

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: the debugging

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (Preeticp) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: Remove 'with'

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (pmacko1) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: yes, $HOME points to current home directory

**Comment by pmacko1** [comment]
@Preeticp could you check for a lang. @amitkrout for the tech updates, please? :)

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (amitkrout) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: It seems pr https://github.com/openshift/odo/pull/2346 is responsible for ```Information prompts for `odo service create and providing possible next step scenarios``` not for ```odo url create commands```. If yes then the proper sentence would be ```Information prompts have been added to the `odo service create` and providing possible next step scenarios.```

@cdrage can you please confirm this ?

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (amitkrout) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: This issue has been closed. please remove this.

**Review by amitkrout — CHANGES_REQUESTED** [review]

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (cdrage) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: Yes, you're correct. Sorry! Remove the URL portion please. Only `service create` was updated.

**Review by cdrage — APPROVED** [review]
LGTM!

/approve

File: modules/openshift-developer-cli-attributes.adoc
Comment (cdrage) [inline modules/openshift-developer-cli-attributes.adoc]: wait what

Comment (cdrage) [reply modules/openshift-developer-cli-attributes.adoc]: We're releasing 1.1.0 along with 4.3, correct? Want to make sure before this merged. Otherwise LGTM.

**Comment by pmacko1** [comment]
Hi @bergerhoffer, can I ask you for a review, please ? Thanks and happy holidays! :)

**Comment by bergerhoffer** [comment]
Hi @pmacko1! I'm sorry, I'm out for the holidays already. Can you ping @peer-review-squad in #team-docs on slack and ask for a review? Happy holidays to you too!

**Comment by Preeticp** [comment]
This PR has been approved by the SMEs and the QE.
It is applicable to the 4.3 branch

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/pod/Pod

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: Noticing that odo as a noun is not in back ticks, while oc as a noun in the docs is. Personally, I like it as you have it with no formatting in {odo-title}, but something to consider for consistency. At some point, we should decide on which to use and add to: https://github.com/openshift/openshift-docs/blob/enterprise-4.2/contributing_to_docs/term_glossary.adoc

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: This section is not rendering in the preview build. I think you need to make this an h2 (or possibly add a module).

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/within/within a

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/developers such as projects, applications and/developers, such as projects, applications, and

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/OpenShift/{product-title}

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/catalog which/catalog, which/
s/deployed on/deployed on a/

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/which/that

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: Remove `////` here

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/have found/find

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/have found/find
s/a bug or/a bug, or
Also, you may want to use {product-title} instead of spelling out OCP, even though it's a UI field.

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/description of the issue/issue description

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/git based/Git-based

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/with `--context` flag/with the `--context` flag/

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/create an/create a/
Also, this sentence is confusing. It sounds like "create a URL route" is an instruction. I think it's missing a comma, so try something like, "When you set an environment variable, create a URL route, and change the source code of the component, the URL route becomes inaccessible."

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/services/Services (for some reason, we are capitalizing Services).

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/OpenShift/{product-title}

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/cluster, `odo/cluster, the `odo/

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/services/Services

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: s/start the port/start port

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [inline cli_reference/openshift_developer_cli/odo-release-notes.adoc]: This sentence is confusing. Suggest: "...to specify the remote port that the debugging agent should on."

**Review by bobfuru — COMMENTED** [review]
Some comments and suggestions. Let us know when you're ready for merge.

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (pmacko1) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: it is not supposed to be rendered. It is commented out (//// text ////). Leaving it here for future reference as per @Preeticp request.

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (pmacko1) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: I'd leave this as is, don't want to spell out OCP here.

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (pmacko1) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: took me a while to decipher ;)
https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#ignoring-markdown-formatting
...but it is a pain to escape all formatting chars, so we good. just fyi :)

**Comment by pmacko1** [comment]
hi @bobfuru thanks a lot for the review! I've revisited the text, could you please give, hopefully last, sanity check, please?

**Comment by pmacko1** [comment]
hi @bobfuru could you take a look, please? :) I'll squash the commits after ack, don't want the comments to disappear/become weird.

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (bobfuru) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: Why not spell it out here?

**Review by bobfuru — COMMENTED** [review]
LGTM @pmacko1

File: cli_reference/openshift_developer_cli/odo-release-notes.adoc
Comment (pmacko1) [reply cli_reference/openshift_developer_cli/odo-release-notes.adoc]: no extra info, redundant (`OpenShift`-short is used throughout the docs so I believe it is ok. And it is not "OCP specific")

**Comment by pmacko1** [comment]
@bobfuru sorry, I've updated the text for "technology preview" to adhere to https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#indicating-technology-preview-features Could you take a last look, please, and merge it?

**Comment by bobfuru** [comment]
LGTM

**Comment by bobfuru** [comment]
@pmacko1 This has been merged and CP to 4.3

**Pull request merged**
